### PR TITLE
appending Content-Type: application/javascript when response is wrapped in callback (JSONP)

### LINKED ISSUE #87: Wrong Content-Type when "callback" query string param is used
When the JSON is wrapped in a callback (JSONP) the content type should be `application/javascript`

It is now being sent as `text/html` which doesn't work when the server also returns `X-Content-Type-Options: nosniff`. (*At least not in Chrome*)

### PR DESCRIPTION
**Pull request by bob-gray**
Fixes #87

### CODE REVIEW

File: src/route.js [hunk 1]
```diff
@@ -22,7 +22,11 @@ function route (req, res, geojson, options) {
     req.query[key] = tryParse(req.query[key])
   })
 
-  if (req.query.callback) req.query.callback = sanitizeCallback(req.query.callback)
+  if (req.query.callback) {
+    req.query.callback = sanitizeCallback(req.query.callback)
+    res.append('Content-Type', 'application/javascript')
```
Comment (dmfenton) [inline src/route.js]: better to use `res.set` here in case content-type is already set

**Comment by dmfenton** [comment]
Thanks for the contribution!

**Comment by bob-gray** [comment]
KoopJS looks like a really cool project! I'm glad I can help a little.

**Comment by rgwozdz** [comment]
Released in [v2.10.2](https://github.com/koopjs/FeatureServer/releases/tag/v2.10.2).  Thanks @bob-gray.

**Pull request merged**